examples/react/turnorder/board.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
Just curious why this file appears to be added in your PR. Perhaps you based it off a commit before https://github.com/nicolodavis/boardgame.io/commit/d50015db218fa89edcf898300a7fcbf687f48317 and the rebase / merge lost something?

Reply:
Oh I see. Militia isn't really a game. It's a small example from the game Dominion. Let's just have TicTacToe and Chess.

Reply:
ok! fixed.